angular/projects/spark-angular/src/lib/directives/sprk-button/sprk-button.directive.spec.ts

```diff
@@ -49,9 +45,8 @@ describe('Spark Button Directive', () => {
   let button3Element: HTMLElement;
   let button4Element: HTMLElement;
   let button5Element: HTMLElement;
-  let button6Element: HTMLElement;
   // TODO: Remove this test is for checking if new spinner directive is being used #3561
-  let button7Element: HTMLElement;
```

Comment:
no longer using this one so I had to rename everything in the test files that were using it.